packages/beacon-node/src/metrics/metrics/lodestar.ts

```diff
@@ -912,7 +912,8 @@ export function createLodestarMetrics(
         name: "validator_monitor_beacon_block_delay_seconds",
         help: "The delay between when the validator should send the block and when it was received",
         labelNames: ["src"],
-        buckets: [0.1, 0.25, 0.5, 1, 2, 5, 10],
+        // we also want other nodes to received our published before 4s so add bucket 3 and 3.5
+        buckets: [0.1, 0.25, 0.5, 1, 2, 3, 3.5, 4, 6, 10],
```

Comment:
```suggestion
        buckets: [0.1, 0.25, 0.5, 1, 2, 3, 4, 6, 10],
```

Removing 3.5 as I don't think it gives us any benefit. From our metrics, we consistently get blocks with 1s delay. Any block sent after 3s will likely not reach enough of the network, the half second doesn't help us.